pkg/engine/states/remote/http/http_backend.go

```diff
@@ -0,0 +1,73 @@
+package http
+
+import (
+	"errors"
+	"strings"
+
+	"github.com/zclconf/go-cty/cty"
+	"kusionstack.io/kusion/pkg/engine/states"
+)
+
+type HTTPBackend struct {
+	HTTPState
+}
+
+func NewHTTPBackend() states.Backend {
```

Comment:
Why don't invoke ConfigSchema() and validates params with Configure() here like the old function NewHTTPState in http_state